staff/miguel-angel-zapatero/cv/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <header>
+        <section>
+            <!-- <hgroup> Por si quiero poner un subtitulo luego con cosas interesantes mias -->
```

Comment:
@maz-e hgroup will be dropped soon, so we recommend you to not use it anymore. Here you have more info: https://www.w3.org/html/wg/wiki/ChangeProposals/hgroup

Reply:
@lexbarba Hi Alex! all hgroup removed, thanks!